tests/boards/ehl_crb/gpio/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.8.2)
+
```

Comment:
I think you are missing `find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})`. See https://lists.zephyrproject.org/g/announce/message/93